design/CreateExperiment.md

```diff
@@ -11,6 +11,23 @@ tuned.
   Stay tuned
 * namespace \
   Stay tuned
+* performance_profile \
+  `performance_profile` is a pre-defined performance objective that is baked into Kruize.
+
+
+`performance_profile` consists of two parts:
+
+* A yaml file defines the metrics to be monitored and the queries used to derive those metrics.
+* Source code that defines the performance objective using the metrics provided
+
+Currently we have the following pre-defined profiles
+
+| Profile Name                           | yaml                                        | Associated Source | Comments                                       |
+|----------------------------------------|---------------------------------------------|-------------------|------------------------------------------------|
+| resource-optimization-openshift        | [resource-optimization-openshift yaml](https://github.com/kruize/autotune/blob/master/manifests/autotune/performance-profiles/resource_optimization_openshift.yaml)        | [source](https://github.com/kruize/autotune/blob/master/src/main/java/com/autotune/analyzer/performanceProfiles/PerformanceProfileInterface/ResourceOptimizationOpenshiftImpl.java)            | This is used for Remote Monitoring Usecase     |
```

Comment:
Also I think, you can mention the name of the same Java file `ResourceOptimizationOpenshiftImpl.java` in both cases to indicate that they both are mapped to the same source

Reply:
Replaced source with file name and added a link to point it